src/utils/transactions.ts

```diff
@@ -49,21 +49,29 @@ export function standardizeSafeTransaction(tx: SafeTransactionDataPartial): Safe
 
 export class SafeTransaction {
   data: SafeTransactionData
-  signatures: Map<string, SafeSignature> = new Map()
+  #signatures: Map<string, SafeSignature> = new Map()
```

Comment:
In safe-react we once run in trouble that we had mixed signatures for lowercase/checksummed owner addresses, IDK if you have protection against this, but I think it would be nice to have :)